### DISCUSSION TITLE
CSS Overrides for different Themes

### DISCUSSION THREAD

**Discussion by GoPro16** [tailwindlabs/tailwindcss#2481] [Help] [answered]
Deciding on what css framework to use next for on one our new projects and need something more flexible vs the base ones such as bootstrap, material etc.

I understand you have the `tailwind.config.js` file that drives css overrides but how extendible is that once you start writing the code and want to overwrite the theming based on say user preferences once logged in.

If this is not a feature and wasn't intended to be I get it, just want to know if this is possible to do in a smooth fashion vs trying to force it upon the framework.

**Comment by adamwathan** [comment, marked as answer]
Best way to handle this right now is to define your color palette using CSS variables. I've got a very old demo repo here:

https://github.com/adamwathan/theming-tailwind-demo

**Reply by GoPro16** [reply]
Awesome, are there any plans to change this in the future?
